Readme.md

```diff
@@ -10,10 +10,11 @@ In addition, a classifier is trained to score the matched occurrences of the con
 ## Data Requirements
 The construction for the automaton requires a SKOS thesaurus represented as a `rdflib` `Graph`.
 Concepts should be related to labels by `skos:prefLabel` or `skos:altLabel`.
-In addition it is assumed that concepts are organized in a hierarchy that includes sub-thesauri.
-Concepts and sub-thesauri have to be distinguishable by `rdf:type`.
-The training of the predictor requires labeled text.
+Concepts have to be identifiable be by `rdf:type`.
+The training of the predictor requires labeled short text.
 Each training sample should be annotated with one or more concepts from the thesaurus.
+Currently the algorithm does not work with long texts,
+i.e., you should provide the titles only or possibly an abstract.
```

Comment:
Vorschlag für vorige und diese Zeile: "Currently the algorithm does not yield satisfactory results for longer texts, i.e., you should work with titles and keywords only, or possibly with abstracts but not with fulltexts."